tools/install_venv_common.py

```diff
@@ -94,10 +94,10 @@ def create_virtualenv(self, no_site_packages=True):
         if not os.path.isdir(self.venv):
             print('Creating venv...', end=' ')
             if no_site_packages:
-                self.run_command(['virtualenv', '-q', '--no-site-packages',
+                self.run_command(['virtualenv', '-p', '/usr/bin/python', '-q', '--no-site-packages',
```

Comment:
How will it handle both cases py2 and py3??

Reply:
Our tempest is not py3 compatible. In case py2 and py3 are installed on the machine, there might be chances that virtual env might pick py3 and our tempest would fail in that case. Hence using only py2 for creating virtual env.